src/style.css

```diff
@@ -0,0 +1,68 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  position: absolute;
+  top: 10px;
+  height: 40px;
+  width: 40px;
+}
```

Comment:
add class to a link and style it
also I think you don't need to positioning here, you can instead utilise margin